### ISSUE TITLE
Float labels inserts label as a select option?

### ISSUE THREAD

**Issue by matthew-dean** [geminilabs/float-labels.js#9]
Given this markup
```html
<label for='contact_country_id>COUNTRY</label>
     <select name="contact[country_id]" id="contact_country_id">
       <option>Select a country</option>
       <option value="1">Afghanistan</option>
```
...once I setup FloatLabels `const floatlabels = new FloatLabels('form', {});`, it will **insert** the word "COUNTRY" as the first selected option in the select, ignoring any other pre-selected country. Why would this be desired behavior and how do I prevent it?

**Comment by pryley** [comment]
The `prioritise` option determines whether to prioritise the label or the placeholder as the floating label. Try this:
```js
const floatlabels = new FloatLabels('form', {
    prioritize: 'placeholder',
});
```

**Comment by pryley** [comment]
This issue has further been addressed with v3.2.0, it should no longer incorrectly insert a placeholder OPTION when one already exists.
